machine/reference/inspect.md

```diff
@@ -61,8 +61,10 @@ For the most part, you can pick out any field from the JSON in a fairly
 straightforward manner.
 
 ```none
+{% raw %}
 $ docker-machine inspect --format='{{.Driver.IPAddress}}' dev
```

Comment:
It looks like that even with `{% raw %}` some component (probably highligh js) is trying to interpret the `{{.Driver.IPAddress}}` part.
I'm getting a JS error on my browser, when previewing `/machine/reference/inspect/` on my browser.

I could fix this error by changing `{{.Driver.IPAddress}}` to `{{Driver.IPAddress}}`. But that's not the Docker syntax.

```
alljs.js:2 Error: [$parse:syntax] Syntax Error: Token '.' not a primary expression at column 1 of the expression [.Driver.IPAddress] starting at [.Driver.IPAddress].
http://errors.angularjs.org/1.3.14/$parse/syntax?p0=.&p1=not%20a%20primary%20expression&p2=1&p3=.Driver.IPAddress&p4=.Driver.IPAddress
    at alljs.js:1
    at Parser.throwError (alljs.js:2)
    at Parser.primary (alljs.js:2)
    at Parser.unary (alljs.js:2)
    at Parser.multiplicative (alljs.js:2)
    at Parser.additive (alljs.js:2)
    at Parser.relational (alljs.js:2)
    at Parser.equality (alljs.js:2)
    at Parser.logicalAND (alljs.js:2)
    at Parser.logicalOR (alljs.js:2)
```